asabiris/orchestration/kafkahandler.py

```diff
@@ -10,36 +10,39 @@
 
 from .emailschema import email_schema
 
-
 #
 
 L = logging.getLogger(__name__)
 
+
 #
 
 
 class KafkaHandler(asab.Service):
-
 	ValidationSchemaMail = fastjsonschema.compile(email_schema)
 
 	# TODO: This is incorrect
 	ValidationSchemaSlack = fastjsonschema.compile({
```

Comment:
do `slack_schema` variable here, same as the email

Reply:
@ateska okay, thanks!